playground/src/pages/CodingEnvironment.tsx

```diff
@@ -15,6 +16,96 @@ async function getExample(example: string) {
 
 function CodingEnvironment() {
 
+  /**
+  * Helper function to check if input contains only numbers.
+  */ 
+   function checkInputField(jsonField: JSON, key: string) {
+
+    const jsonInput = jsonField[key as keyof typeof jsonField];
+
+    if (!Array.isArray(jsonInput)) {
+
+      const errorMessage = `If you use ${key},
+      it must be an array of numbers.`;
+
+      toast.error(errorMessage);
+
+      setOutput(errorMessage);
+
+      return false;
+
+    }
+
+    if (Object.values(jsonInput).length === 0 || Object.values(jsonInput).some(isNaN)) {
+
+      const errorMessage = `If you use ${key}, 
+      it must contain at least one number, 
+      and it can only contain numbers.`;
+
+      toast.error(errorMessage);
+
+      setOutput(errorMessage);
+
+      return false;
+    }
+
+    return true;
+
+    }
+
+
+  /**
+  * We check the inputs and return true or false. We allow: 
+  * - an empty input, and
+  * - a valid JSON object containing stack_init or advice_tape (or both) 
+  * if the values are numbers
+  */ 
+  function checkInputs(jsonString: string) {
+
+    if (jsonString === "") {
+      return true;
+    }
+
+    let jsonInput!: JSON;
+
+    try {
+
+      jsonInput = JSON.parse(jsonString);
+
+    }
+    catch (e) {
+
+      const errorMessage = `Miden VM Inputs need to be 
+a valid JSON object.`;
+
+      toast.error(errorMessage);
+
+      setOutput(errorMessage);
+
+      return false;
+
+    }
+
+    if (!Object.keys(jsonInput).includes("stack_init") && !Object.keys(jsonInput).includes("advice_tape")) {
```

Comment:
humm, this doesn't seem to be caugthing the case with only `advice_tape`:

```
{
    "advice_tape": ["0"]
}
```

The error I get is:

![image](https://user-images.githubusercontent.com/310139/219375699-f1649794-2ee6-4d73-bf8b-ad0fe4c637a8.png)

Reply:
damn good catch, I need to create tests for all cases. Thanks. 

I need to change that in the Rust file, we want to allow Inputs with only `advice_tape`